static/data/hierarchy_statsvar.json

```diff
@@ -6,7 +6,7 @@
   "c": 1784,
   "cd": [
    {
-    "sv": "Count_Person",
+    "sv": "TotalPopulation",
```

Comment:
Note this will change back to Count_Person very soon. I will let you know then, so need to generate the json again.

Reply:
Cool. For now, I added one line to convert it.